examples/rl_example.py

```diff
@@ -1,178 +1,258 @@
 import logging
 import os
 import sys
-import time
 from typing import Callable
 
 import torch
+import time
 from torch_memory_saver import torch_memory_saver
-from examples.util import print_gpu_memory_gb
+from util import print_gpu_memory_gb
 
 logging.basicConfig(level=logging.DEBUG, stream=sys.stdout)
 
+# Define the size of a large tensor for simulating KV cache
+# Size: 5 * 100,000,000 * 4 bytes = 2GB
 dummy_tensor_size = (5, 100_000_000,)
 
 
 def _ptr(x):
+    """
+    Get the virtual address of a tensor.
+    
+    Virtual Address is the address that the process sees, not the actual physical memory address.
```

Comment:
oh I see, are you using this example py file as a kind of tutorial to tell readers how it is implemented? If so then the comments may be ok (otherwise it would be great to remove all or most comments), and maybe add a comment at the start of the whole file to tell readers this is a tutorial